### PR TITLE
LET-268

### PR DESCRIPTION
**Pull request by agnessa**
https://vizzuality.atlassian.net/browse/LET-268?atlOrigin=eyJpIjoiMTY1NmZkYmFmM2RkNDBjMjkxYWNhYWVmOTk0MWZjMDkiLCJwIjoiaiJ9

This is so that we have error reporting where we rescue exceptions. Also removed the test exception controller.

### CODE REVIEW

File: backend/app/jobs/translate_job.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ class TranslateJob < ApplicationJob
   rescue_from(ActiveRecord::RecordNotFound) do |exception|
     Rails.logger.error(exception)
     Rails.logger.error(Rails.backtrace_cleaner.clean(exception.backtrace).join("\n"))
-    # TODO: handle exception tracking
+    Google::Cloud::ErrorReporting.report exception
```
Comment (martintomas) [inline backend/app/jobs/translate_job.rb:7]: I remember that I have added some hooks for BackgroundJob Events to ApplicationJob and they also store exception to db. Maybe that reporting should be added there and all these `rescue_from ` and `handle_exception ` could be removed so we deal with exceptions at one place for all current and future jobs.

Comment (martintomas) [reply backend/app/jobs/translate_job.rb:7]:
```
def store_exception_at_event
    yield
  rescue Exception => e # rubocop:disable Lint/RescueException
    upsert_event self, status: :crashed, message: e.backtrace
    raise e
  end
```

**Review by martintomas — APPROVED** [review]

File: backend/app/jobs/translate_job.rb (hunk 1 above)
Comment (tsubik) [reply backend/app/jobs/translate_job.rb:7]: I agree we can move it there.

**Comment by tsubik** [comment]
@agnessa please also add this to `api/errors.rb` concern.

File: backend/app/jobs/translate_job.rb (hunk 1 above)
Comment (agnessa) [reply backend/app/jobs/translate_job.rb:7]: OK! Thanks!

Comment (agnessa) [reply backend/app/jobs/translate_job.rb:7]: Oh but actually in that case we do a `raise` - that will automatically report

Comment (tsubik) [reply backend/app/jobs/translate_job.rb:7]: Ohh, ok. Then, if we still need to have the clean error messages in the logger (not sure if there is any need of that now, if we have error management), we can move Rails.logger part.

Comment (tsubik) [reply backend/app/jobs/translate_job.rb:7]: We can keep it, dunno :)

Comment (tsubik) [reply backend/app/jobs/translate_job.rb:7]: Or actually, raising exception will get a good log as well, so we can just remove the whole thing, lol.

**Comment by agnessa** [comment]
Added to the controller concern and removed from Translate job and service - it will be handled in the parent job.

**Pull request merged**